lib/msal-node/src/cache/serializer/SerializerTypes.ts

```diff
@@ -3,14 +3,18 @@
  * Licensed under the MIT License.
  */
 
-import { StringDict, AccountCache, IdTokenCache, AccessTokenCache, RefreshTokenCache, AppMetadataCache } from "@azure/msal-common";
+import {AccountCache, IdTokenCache, AccessTokenCache, RefreshTokenCache, AppMetadataCache } from "@azure/msal-common";
+
+export type Dict = {
```

Comment:
`Dict` is not specific enough, can this be something like StringDictionary?

Reply:
+1

Reply:
Updated the usage and got rid of `Dict`